src/classes/groundplaces.ts

```diff
@@ -307,17 +371,26 @@ export class GroundPlacesController {
 
     // Update the place with the new StopCluster parent
     this.storageService.replacePlace(newStopClusterParent);
+
+    if (shouldCreateDiff === CreateDiff.TRUE) {
```

Comment:
I know it was the previous wording, and i dont know what the others think, but personally i dont like the wording `groudPlaceDiff`. I think of it more like an `groundPlaceActionHistory` or `groundPlaceChanges`. A to me a diff file would recap the differences between two files.
i'd view `shouldCreateDiff` to be smtg like `shouldAddToHistory`, or `shouldSaveChange`

@Simobain @rimiti

Reply:
Pour moi c'est un peu difficile à donner un nom car c'est : 
- A la fois un fichier qui va contenir les actions qui ont été faites sur les GroundPlaces
- Mais également un fichier qui peut envoyer des instructions à effectuer sur les GroundPlaces

Peut être chercher du côté de `ActionHistory` si on souhaite le renommer ?

Reply:
that would make more sense to me. what do you think @rimiti @Simobain ?

Reply:
I agreed with @Theo-Le-Donne. The wording should more explicit.

Reply:
groundPlaceActionHistory ✅